From 6e320c98b7a41369b32f77d5a921b7b83f7d694a Mon Sep 17 00:00:00 2001
From: Tamas Ban <tamas.ban@arm.com>
Date: Wed, 21 Feb 2024 11:40:21 +0100
Subject: [PATCH 848/954] refactor(rss): rename all 'rss' files to 'rse'

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 56f27d8f793bf523fa414335a99603ff5f5104bb refactor(rss): rename all 'rss' files to 'rse']
Signed-off-by: Tamas Ban <tamas.ban@arm.com>
Change-Id: I6cfca1d67e246d5079f683241021ed039cc27f74
(cherry picked from commit 955116982f5bfb6803b1961eda5fa24ab7886419)
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/443840
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 drivers/arm/{rss/rss_comms.c => rse/rse_comms.c}                  | 0
 drivers/arm/{rss/rss_comms.mk => rse/rse_comms.mk}                | 0
 .../arm/{rss/rss_comms_protocol.c => rse/rse_comms_protocol.c}    | 0
 .../arm/{rss/rss_comms_protocol.h => rse/rse_comms_protocol.h}    | 0
 .../rse_comms_protocol_common.h}                                  | 0
 .../rss_comms_protocol_embed.c => rse/rse_comms_protocol_embed.c} | 0
 .../rss_comms_protocol_embed.h => rse/rse_comms_protocol_embed.h} | 0
 .../rse_comms_protocol_pointer_access.c}                          | 0
 .../rse_comms_protocol_pointer_access.h}                          | 0
 include/drivers/arm/{rss_comms.h => rse_comms.h}                  | 0
 10 files changed, 0 insertions(+), 0 deletions(-)
 rename drivers/arm/{rss/rss_comms.c => rse/rse_comms.c} (100%)
 rename drivers/arm/{rss/rss_comms.mk => rse/rse_comms.mk} (100%)
 rename drivers/arm/{rss/rss_comms_protocol.c => rse/rse_comms_protocol.c} (100%)
 rename drivers/arm/{rss/rss_comms_protocol.h => rse/rse_comms_protocol.h} (100%)
 rename drivers/arm/{rss/rss_comms_protocol_common.h => rse/rse_comms_protocol_common.h} (100%)
 rename drivers/arm/{rss/rss_comms_protocol_embed.c => rse/rse_comms_protocol_embed.c} (100%)
 rename drivers/arm/{rss/rss_comms_protocol_embed.h => rse/rse_comms_protocol_embed.h} (100%)
 rename drivers/arm/{rss/rss_comms_protocol_pointer_access.c => rse/rse_comms_protocol_pointer_access.c} (100%)
 rename drivers/arm/{rss/rss_comms_protocol_pointer_access.h => rse/rse_comms_protocol_pointer_access.h} (100%)
 rename include/drivers/arm/{rss_comms.h => rse_comms.h} (100%)

diff --git a/drivers/arm/rss/rss_comms.c b/drivers/arm/rse/rse_comms.c
similarity index 100%
rename from drivers/arm/rss/rss_comms.c
rename to drivers/arm/rse/rse_comms.c
diff --git a/drivers/arm/rss/rss_comms.mk b/drivers/arm/rse/rse_comms.mk
similarity index 100%
rename from drivers/arm/rss/rss_comms.mk
rename to drivers/arm/rse/rse_comms.mk
diff --git a/drivers/arm/rss/rss_comms_protocol.c b/drivers/arm/rse/rse_comms_protocol.c
similarity index 100%
rename from drivers/arm/rss/rss_comms_protocol.c
rename to drivers/arm/rse/rse_comms_protocol.c
diff --git a/drivers/arm/rss/rss_comms_protocol.h b/drivers/arm/rse/rse_comms_protocol.h
similarity index 100%
rename from drivers/arm/rss/rss_comms_protocol.h
rename to drivers/arm/rse/rse_comms_protocol.h
diff --git a/drivers/arm/rss/rss_comms_protocol_common.h b/drivers/arm/rse/rse_comms_protocol_common.h
similarity index 100%
rename from drivers/arm/rss/rss_comms_protocol_common.h
rename to drivers/arm/rse/rse_comms_protocol_common.h
diff --git a/drivers/arm/rss/rss_comms_protocol_embed.c b/drivers/arm/rse/rse_comms_protocol_embed.c
similarity index 100%
rename from drivers/arm/rss/rss_comms_protocol_embed.c
rename to drivers/arm/rse/rse_comms_protocol_embed.c
diff --git a/drivers/arm/rss/rss_comms_protocol_embed.h b/drivers/arm/rse/rse_comms_protocol_embed.h
similarity index 100%
rename from drivers/arm/rss/rss_comms_protocol_embed.h
rename to drivers/arm/rse/rse_comms_protocol_embed.h
diff --git a/drivers/arm/rss/rss_comms_protocol_pointer_access.c b/drivers/arm/rse/rse_comms_protocol_pointer_access.c
similarity index 100%
rename from drivers/arm/rss/rss_comms_protocol_pointer_access.c
rename to drivers/arm/rse/rse_comms_protocol_pointer_access.c
diff --git a/drivers/arm/rss/rss_comms_protocol_pointer_access.h b/drivers/arm/rse/rse_comms_protocol_pointer_access.h
similarity index 100%
rename from drivers/arm/rss/rss_comms_protocol_pointer_access.h
rename to drivers/arm/rse/rse_comms_protocol_pointer_access.h
diff --git a/include/drivers/arm/rss_comms.h b/include/drivers/arm/rse_comms.h
similarity index 100%
rename from include/drivers/arm/rss_comms.h
rename to include/drivers/arm/rse_comms.h
-- 
2.53.0

